web/components/inputs/FcDatePicker.vue

```diff
@@ -9,33 +9,35 @@
       offset-y
       transition="scale-transition">
       <template v-slot:activator="{ attrs, on: onMenu }">
```

Comment:
can we do a similar `attrs: attrsMenu` thing here?  that would help make it clear that these are a11y attributes specifically related to the `<v-menu>` activator slot